packages/flutter_tizen/lib/src/rendering/platform_view.dart

```diff
@@ -118,6 +118,7 @@ class RenderTizenView extends PlatformViewRenderBox {
     do {
       targetSize = size;
       _currentTextureSize = await _viewController.setSize(targetSize);
+      _viewController.setOffset(localToGlobal(Offset.zero));
```

Comment:
What's the relationship between the resize and offset operations? Why should they performed together? Why didn't you add a post frame callback just like `RenderAndroidView`?

Reply:
> What's the relationship between the resize and offset operations? Why should they performed together?

`create(size, offset)` takes size and offset as arguments. However, when `_viewController.setSize(targetSize)` is called, only size is used to create a platform view(`create(size)`).  After the platform view is created, the platform view is notified of the offset.
> Why didn't you add a post frame callback just like RenderAndroidView?

The use of postframe callback has been confirmed to work as I intended. I'll apply the code. Thank you.

Reply:
> `create(size, offset)` takes size and offset as arguments. However, when `_viewController.setSize(targetSize)` is called, only size is used to create a platform view(`create(size)`). After the platform view is created, the platform view is notified of the offset.

This doesn't answer my question. I was asking why you called `setOffset` inside the method (`_sizePlatformView`) whose role is to "resize" the platform view but not to set its offset.